scripts/quests/outlands/An_Undying_Pledge.lua

```diff
@@ -0,0 +1,132 @@
+-----------------------------------
+-- An Undying Pledge
+-----------------------------------
+-- Log ID: 5, Quest ID: 149
+-- Stray Cloud : !pos -18 1 -20 252
+-- qm5         : !pos 133 -9 221 176
+-----------------------------------
+local ssgID = zones[xi.zone.SEA_SERPENT_GROTTO]
```

Comment:
Just a small nitpick here, but can you style that similar to other formats? (with --- block under the local)